src/Tooltip.vue

```diff
@@ -4,9 +4,9 @@
     -->
     <transition :name="effect">
       <div ref="popover" v-if="show" style="display:block;"
-        :class="['tooltip',placement]"
+        :class="['tooltip',tooltipPlacementClass, 'show']"
```

Comment:
spacing after the comma:

`:class="['tooltip', tooltipPlacementClass, 'show']"`